ci/Dockerfile.centos

```diff
@@ -52,5 +52,6 @@ RUN ln -sf /usr/bin/cmake3 /usr/bin/cmake
 # Install the Cloud Bigtable emulator and the Cloud Bigtable command-line
 # client.  They are used in the integration tests.
 WORKDIR /var/tmp/install/cbt-components
-RUN curl https://sdk.cloud.google.com | bash -s -- --disable-prompts --install-dir=/usr/local
+RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-187.0.0-linux-x86_64.tar.gz
```

Comment:
Consider adding a comment here with the previous version of the install command? This will be out-of-date soon since it hard-codes a specific version of gcloud, but we want to always test with the latest and greatest.

I imagine we will want to revert to the previous version; it's obviously possible to examine the git history to see what it was, but keeping the old version in as a comment with a TODO to revert to it may be helpful.

Just a thought.

(Similar comment in the other 3 files where the same steps occur, naturally.)